Tests/Integration/AbstractIntegrationTest.php

```diff
@@ -186,12 +211,39 @@ protected function handleResourceRequest(ServerRequestInterface $serverRequest):
         return $serverRequest;
     }
 
+    protected function handleAuthorizeRequest(ServerRequestInterface $serverRequest, $approved = true): array
```

Comment:
@ajgarlag Looking at method name now, it seems unclear compared to handleAuthorizationRequest. I would suggest renaming this method to ie. `handleAuthorizationCodeGrantRequest` and @spideyfusion or I can rename `handleAuthorizationRequest` to more clear name as well. What do you think?

Reply:
This method will be used to test implicit grant too, so the `handleAuthorizationCodeGrantRequest` name is not very good.

My first option was to rename `handleAuthorizationRequest` to `handleTokenRequest`, and name the new method `handleAuthorizationRequest` but I discarded it to avoid a BC break. But if you think that this method can be renamed, this will be my preferred option. @alenpokos What do you think?

Reply:
@alenpokos Can you confirm me if renaming the `handleAuthorizationRequest` is an acceptable solution? Are you going to rename it in another PR, or do you prefer to include the change in this one?

Reply:
@ajgarlag Sry for wait. Yes, the name sounds good and I think it is ok to rename it in this MR as this is where we noticed the need to change.